pkgs/top-level/all-packages.nix

```diff
@@ -19145,4 +19145,6 @@ with pkgs;
   undaemonize = callPackage ../tools/system/undaemonize {};
 
   houdini = callPackage ../applications/misc/houdini {};
+
+  duti = callPackage ../os-specific/darwin/duti {};
```

Comment:
Actually should it be in the "Darwin" namespace? Not sure if that's only supposed to be core stuff though.